http_handlers.go

```diff
@@ -46,7 +46,7 @@ func RegisterEchoHandlers(svcHandler ServiceHandler, e *echo.Echo) {
 			return c.String(http.StatusInternalServerError, err.Error())
 		}
 		bytes, err := validateRequest(svc, c)
-		if err != nil {
+		if bytes == nil { // validateRequest failed and sent the response
```

Comment:
Maybe `if bytes == nil || err != nil { ...`, just to cover all corner cases?

Reply:
fixed in force-push